packages/components/src/SubHeaderBar/InputTitleSubHeader/InputTitleSubHeader.test.js

```diff
@@ -69,6 +69,12 @@ describe('TitleSubHeader', () => {
 		expect(wrapper.getNode().props.className).toEqual('tc-subheader-details-text-subtitle');
 		expect(wrapper.getNode().props.children).toEqual('mySubTitle');
 	});
+	it('should render with no edit pencil', () => {
```

Comment:
The default state is not editable. Can you remove the `editable` from other tests ?
The classical render snapshot will cover the no-edit state

In general, test the default, then test the special cases in specific tests